### PR TITLE
Add broadcast support

### PR DESCRIPTION
**Pull request by boraozgen**
For issue #195 

@biancode I covered the serial use cases but I don't know much about the TCP mode, or if this breaks anything there...

BTW how do you run the tests? npm test results in many errors for me..

### CODE REVIEW

File: src/core/modbus-client-core.js [hunk 1]
```diff
@@ -236,8 +242,16 @@ de.biancoroyal.modbus.core.client.writeModbusByFunctionCodeFive = function (node
...
+    if (node.client.getID() === 0) {
+      const resp = {
+        address: parseInt(msg.payload.address),
+        value: parseInt(msg.payload.value)
+      }
+      coreClient.activateSendingOnSuccess(node, cb, cberr, resp, msg)
+    } else {
+      coreClient.activateSendingOnFailure(node, cberr, err, msg)
+      node.modbusErrorHandling(err)
+    }
```
Comment (boraozgen) [inline src/core/modbus-client-core.js:254]: Some clarification here: modbus-serial does return an error when a response is not received, even if unitid=0. See [this issue](https://github.com/yaacov/node-modbus-serial/issues/97) Therefore I added the check in the error handler..

**Comment by boraozgen** [comment]
Also addresses #188 .. gotta test with TCP

**Comment by biancode** [comment]
@boraozgen I added some information in the readme yesterday see For Developers

./clean.sh

You could use Visual Studio Code to run and debug tests too ;)

**Comment by biancode** [comment]
@boraozgen did the test work for you? I'll have a look over the weekend ;)

**Comment by boraozgen** [comment]
Nope unfortunately ./clean.sh broke the environment for me. Some "bindings" were missing afterwards. I had to remove node_modules and do npm install again. Didn't have the time to try it again afterwards.

**Comment by biancode** [comment]
@boraozgen could you please send the request to develop, because it has issues on your system to test?

**Pull request closed without merging**